mantidimaging/gui/windows/spectrum_viewer/view.py

```diff
@@ -72,3 +76,10 @@ def try_to_select_relevant_normalise_stack(self, name: str) -> None:
 
     def get_normalise_stack(self) -> Optional['UUID']:
         return self.normaliseStackSelector.current()
+
+    def get_csv_filename(self) -> Optional[Path]:
```

Comment:
Just a thought, but if we ever wanted to re-use the CSV output functionality then it's likely we would need to replicate this method in other views. Would there be any benefit to making this a static method in the `CSVOutput` class instead?

Reply:
I don't think it belongs in core because it use Qt.

If we started using it more places BaseMainWindowView might be a better place. There might be a bit more that can be shared between the windows.

Reply:
Yes, I wondered about that but then checked core and it looks like we're using Qt in some of the operations there. If the intention is actually to keep Qt out of core, though, then better off leaving this where it is.